app/assets/javascripts/global_listeners.js

```diff
@@ -64,16 +74,27 @@ $(function() {
       };
     });
 
-  $(".menu-toggle").click(function(e) {
+  $("#sidebar-wrapper").mouseenter(function() {
```

Comment:
I think you can use underscore's [debounce](http://underscorejs.org/#debounce) here instead of implementing it by hand 😄